Lexplorer/Components/HomepageOverview.razor

```diff
@@ -63,23 +73,35 @@
             <MudCard>
                 <MudCardContent>
                     <MudText Typo="Typo.h6" Align="Align.Center">NFT Mint Count</MudText>
-                    <MudText Typo="Typo.body1" Align="Align.Center">@blockData?.data?.proxy?.nftMintCount.ToString("N0")</MudText>
+                    <MudText Typo="Typo.body1" Align="Align.Center">
+                        @blockData?.data?.proxy?.nftMintCount.ToString("N0") 
+                        <MudText Color="Color.Primary">+@dailyCount.NFTMintCount.ToString("N0") in 24 hours</MudText>
+                    </MudText>
                 </MudCardContent>
             </MudCard>
         </MudItem>
         <MudItem xs="12" sm="12" lg="4">
             <MudCard>
                 <MudCardContent>
                     <MudText Typo="Typo.h6" Align="Align.Center">NFT Trade Count</MudText>
-                    <MudText Typo="Typo.body1" Align="Align.Center">@blockData?.data?.proxy?.tradeNFTCount.ToString("N0")</MudText>
+                    <MudText Typo="Typo.body1" Align="Align.Center">
+                        @blockData?.data?.proxy?.tradeNFTCount.ToString("N0") 
+                        @if(dailyCount.TradeNFTCount > 0)
+                        {
+                                <MudText Color="Color.Primary">+@dailyCount.TransactionCount.ToString("N0") in 24 hours</MudText>
+                        }
```

Comment:
Why is this - and only this - hidden if 0?

Reply:
As of now, this and only this is the only one where there are no traction... So I thought it would be unecessary to show "+0 in 24 hours" all the time.

Reply:
I see your point and it's a valid one. I will remove the check and even if it says 0 in 24 hours, this will soon be different (we hope) :)